Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs

```diff
@@ -302,12 +344,27 @@ private Operation CreateDeleteByIdOperation(OpenApiMetadataPathsResource openApi
                     OpenApiMetadataDocumentHelper.CreateIdParameter(),
                     CreateIfMatchParameter("DELETE from removing")
                 },
-                responses = OpenApiMetadataDocumentHelper.GetWriteOperationResponses(HttpMethod.Delete)
+                responses = responses
             };
         }
 
         private Operation CreateDeletesOperation(OpenApiMetadataPathsResource openApiMetadataResource)
         {
+            var responses = OpenApiMetadataDocumentHelper.GetReadOperationResponses(
+                _pathsFactoryNamingStrategy.GetResourceName(openApiMetadataResource, ContentTypeUsage.Readable),
+                true, true);
+
+            if (_apiSettings.IsFeatureEnabled(ApiFeature.Publishing.GetConfigKeyName()))
+            {
+                responses.Add(
+                    "405",
```

Comment:
This is a read operation, and shouldn't return a 405 ever.  This is for retrieving the deleted items from the tombstone table.